### DISCUSSION TITLE
Error in using advanced restart

### DISCUSSION THREAD

**Discussion by DYLDYLDYL** [idaholab/moose#22941] [Q&A General] [answered]
Hi!

I want to use advanced restart.  But when I  set file = _out.e, it will still start at time step 1. So I wonder how to set file and restart_file_base.
```
[Mesh]
  file = out_cp/0010_mesh.cpr    
  distribution = serial    ## it shows "unused parameter 'Mesh/distribution' "
[]
[Problem]
  restart_file_base = out_cp/LATEST 
[]
```

Thanks in advance for your help!

**Comment by GiudGiud** [comment, marked as answer]
Hello

You should be able to use the original mesh file in [Mesh] and then just set the restart_file_base to LATEST and get the latest time step.

Note that it will start counting at time step 1 regardless. But the initial condition will be from the latest in the file

Guillaume

**Reply by DYLDYLDYL** [reply]
Thanks！
I use the original mesh file and restart_file_base = test_out/0008, it shows 'Unable to open file test_out/0008.rd-0'.
If restart_file_base = test_out, it shows 'Unable to open file test_out.rd-0'.
test_out.e is my ofiginal output, it has run 8 time steps.

**Reply by GiudGiud** [reply]
This is why it s easiest to just specify latest after the folder name.

**Reply by GiudGiud** [reply]
If you dont want to use latest, this is the syntax
  restart_file_base = system_1d_checkpoint_cp/1179
system_1d_checkpoint_cp is the folder where the restart files got saved

**Reply by DYLDYLDYL** [reply]
Thanks! 
Is 1179 a time step？

**Reply by GiudGiud** [reply]
yes. 1179 was the timestep of interest for my simulation

**Reply by DYLDYLDYL** [reply]
Thanks！
